### PR TITLE
Added Filters

### PR DESCRIPTION
**Pull request by zbarnz**
- Added filters to participant `getMeeting` controller
- Added filter tests
- Removed non root directory `.npmrc` files because they aren't needed as of `npm@8.5.0` and were causing issues with my terminal
- Removed a random console log in Nav

### CODE REVIEW

**Review by Dahvd — APPROVED** [review]

File: api/test/fakes/meeting.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 const ObjectId = require('mongoose').Types.ObjectId;
 
 const meeting = ( opts ) => {
+  const randomName = ( Math.random() + 1 ).toString( 36 ).substring( 7 ); //rip faker
```
Comment (thudsonbu) [inline api/test/fakes/meeting.js:4]: Somebody else took over faker, its now at `@faker-js/faker`. Can you replace the faker in the root directory dev dependency with the new version?

**Review by Dahvd — APPROVED** [review]

File: api/test/fakes/meeting.js (hunk 1 above)
Comment (zbarnz) [reply api/test/fakes/meeting.js]: @thudsonbu done

**Review by Dahvd — APPROVED** [review]

**Review by Dahvd — APPROVED** [review]
make sure to fake out tom with faker.js

**Review by thudsonbu — CHANGES_REQUESTED** [review]
I am actually thinking about creating a new meeting index route at `api/meetings`. It will fetch all of the meetings that a user owns or is a participant of. I should have been a more specific. It will be pretty similar to this though, the query will just be quite a bit more involved. 

I would just start with a pr that creates the new route and aggregates all of the meetings a user owns or is a participant of. We can add other filters as we go. It's possible to do this in a single aggregation but it is a kinda complicated so lmk if you need more direction on how that would work and we can hop on a call/live share.

One other thing to note is that you should get the user's email and `_id` from `req.credentials`. That ensures that a bad actor doesn't fabricate a request with someone else's id as a query param.

**Comment by zbarnz** [comment]
> I would just start with a pr that creates the new route and aggregates all of the meetings a user owns or is a participant of.

I think both of these should have their own menu. I imagine it being like sent emails vs received emails. I cant imagine where we would need them both returned in the same query

**Comment by thudsonbu** [comment]
> > I would just start with a pr that creates the new route and aggregates all of the meetings a user owns or is a participant of.
> 
> I think both of these should have their own menu. I imagine it being like sent emails vs received emails. I cant imagine where we would need them both returned in the same query @thudsonbu

The main benefits of using a single route are that we wont need to duplicate the filtering logic and we can search all of a users meetings regardless of if they were the participant or the owner. The second part is important because it might not be easy to remember who was the original creator of a meeting. I think it would be weird if when you search by a meeting tag you can only either search the meetings you own or the meetings you are a participant of. I don't think that is what a user would expect.

**Pull request closed without merging**